tfe/data_source_outputs.go

```diff
@@ -18,14 +17,6 @@ type dataSourceOutputs struct {
 	tfeClient *tfe.Client
 }
 
-var stderr *os.File
-
-func init() {
-	// There is a issue that occurs when the plugin-go Serve function is used that
-	// causes os.Stderr to be overwritten. There is a fix being worked on for this.
-	stderr = os.Stderr
-}
```

Comment:
Did you verify that the fix was in place in plugin-go for this?

Reply:
The flag here is that `var stderr *os.File` is unused. We chatted about this briefly back in Oct and @nfagerlund offered some context around this block of code:

```
last place stderr occurred anywhere in this repo was right before we switched to using go modules 
and removed all the vendored library code
```

Am I missing any additional context?

Reply:
I couldn't sort out how to trace this vague comment back to a specific issue (my ace in the hole was to ask Omar, but I think he couldn't remember either). But even if the issue still existed, we're not ever _doing_ anything with that variable after setting it, so it doesn't seem likely that this would, on its own, mitigate said issue. 

[This is the only PR](https://github.com/hashicorp/terraform-plugin-go/pull/93) in terraform-plugin-go that seems relevant, but again, identifying the exact issue doesn't seem critical if we already weren't actually _doing_ anything to work around it.